### PR TITLE
Add a helper script for preparing the release

### PR DESCRIPTION
**Pull request by marquiz**
For now, it just updates the deployment files image tag and pull policy.

### CODE REVIEW

**Review by fmuyassarov — COMMENTED** [review]
Looks good. Can you make the release-helper.sh executable?

**Comment by marquiz** [comment]
> Can you make the release-helper.sh executable?

Woops, fixed

File: docs/conf.py [hunk 1]
```diff
@@ -107,6 +107,7 @@ def gomod_versions(modules):
 
 mod_versions = gomod_versions(['github.com/intel/goresctrl'])
 myst_substitutions = {
+    'version': version,
     'golang_version': mod_versions['golang'],
     'goresctrl_version': mod_versions['github.com/intel/goresctrl']
 }
```
Comment (fmuyassarov) [inline docs/conf.py:113]: Does this work @marquiz ?

Comment (marquiz) [reply docs/conf.py]: Argh, messed up with some testing 🤯 Thanks for noting, removed

**Review by fmuyassarov — APPROVED** [review]
LGTM

**Pull request merged**